src/watertap_contrib/seto/solar_models/surrogate/trough/test_trough_surrogate.py

```diff
@@ -113,105 +99,142 @@ def test_build(self, trough_frame):
-        m = trough_frame
-        assert m.fs.trough.heat_load.bounds == tuple([100, 1000])
+    def test_surrogate_variable_bounds(self, trough_large_heat_load):
+        m = trough_large_heat_load
+        assert m.fs.trough.heat_load.bounds == tuple([100, 500])
         assert m.fs.trough.hours_storage.bounds == tuple([0, 26])
 
     @pytest.mark.component
-    def test_create_rbf_surrogate(self, trough_frame):
-        expected_heat_annual_test = [
-            8.034e8,
-            3.505e8,
-            3.668e8,
-            3.814e8,
-            3.946e8,
-            4.065e8,
-            4.175e8,
-            4.275e8,
-            4.366e8,
-            4.445e8,
-        ]
-        expected_electricity_annual_test = [
-            7.176e6,
-            1.301e6,
-            1.302e6,
-            1.303e6,
-            1.304e6,
-            1.305e6,
-            1.306e6,
-            1.307e6,
-            1.308e6,
-            1.308e6,
-        ]
-        m = trough_frame
-        data = get_data()
+    def test_create_rbf_surrogate_large(self, trough_large_heat_load):
+        m = trough_large_heat_load
+        data = get_data(m.fs.trough.heat_load.bounds)
         test_surrogate_filename = os.path.join(
             os.path.dirname(__file__), "test_surrogate.json"
         )
         m.fs.trough._create_rbf_surrogate(
-            data_training=data["training"], output_filename=test_surrogate_filename
+            output_filename=test_surrogate_filename
         )
         assert os.path.getsize(test_surrogate_filename) > 1e4
         os.remove(test_surrogate_filename)
         assert isinstance(m.fs.trough.rbf_surr, PysmoSurrogate)
         test_output = m.fs.trough.rbf_surr.evaluate_surrogate(data["validation"])
-        assert list(test_output["heat_annual"]) == pytest.approx(
-            expected_heat_annual_test, 1e-3
+
+        surrogate_scaling = TroughSurrogateData.surrogate_scaling_dict[tuple(m.fs.trough.heat_load.bounds)]
+        expected_heat_annual_test = data['validation']['heat_annual'] * surrogate_scaling
+        expected_electricity_annual_test = data['validation']['electricity_annual'] * surrogate_scaling
+        assert list(test_output["heat_annual_scaled"]) == pytest.approx(
+            expected_heat_annual_test, 1e-2
         )
-        assert list(test_output["electricity_annual"]) == pytest.approx(
-            expected_electricity_annual_test, 1e-3
+        assert list(test_output["electricity_annual_scaled"]) == pytest.approx(
+            expected_electricity_annual_test, 5e-2
         )
 
     @pytest.mark.component
-    def test_validation(self, trough_frame):
+    def test_validation_large(self, trough_large_heat_load):
 
-        m = trough_frame
-        data = get_data()
+        m = trough_large_heat_load
+        data = get_data(m.fs.trough.heat_load.bounds)
         heat_annual_list = []
         electricity_annual_list = []
 
+        solver = SolverFactory("ipopt")
```

Comment:
@MuktaHardikar I recall you saying that these revisions made the model more stable and able to solve under previous conditions where the model was difficult to solve. I thought that you were referring to using "ipopt-watertap" rather than "ipopt". Could you clarify whether solver convergence issues still occur with "ipopt-watertap"?

Reply:
I've been using mainly ipopt-watertap but haven't checked with ipopt in the flowsheet. Previously, without the scaling in the surrogate, the model depended a lot on scaling in the flowsheet to solve. With the updates, the models default scaling is pretty good for most cases and I didn't have convergence issues with the ipopt-watertap solver.

Reply:
So technically, if we delete any occurences of `SolverFactory("ipopt")` in this PR, checks should still pass. I thought I had resolved this issue in a previous PR but had overlooked occurrences of `SolverFactory("ipopt")`. Hopefully, this PR will still pass when we remove.